corehq/apps/userreports/examples/examples.md

```diff
@@ -442,6 +442,58 @@ Note that the test must also change in this scenario.
 }
 ```
 
+## Evaluator Examples
+
+### Age in years to age in months (simple case)
+
+```json
+{
+    "type": "evaluator",
+    "statement": "30.4 * age_in_years",
+    "context_variables": {
+        "age_in_years": 5,
```

Comment:
I think this would be better if the expression was pulling `age_in_years` from a case property